qbx_consumables/server.lua

```diff
@@ -120,6 +143,54 @@ lib.callback.register('consumables:server:usedItem', function(source, item)
 end)
 
 -- Added for ox_inv until I make a proper qbx bridge
-RegisterNetEvent('consumables:server:addThirst', addThirst)
 
-RegisterNetEvent('consumables:server:addHunger', addHunger)
+--- @deprecated confusing name
+RegisterNetEvent('consumables:server:addThirst', function (amount)
+    setThirst(source, amount)
+end)
+--- @deprecated confusing name
+RegisterNetEvent('consumables:server:addHunger', function (amount)
+    setHunger(source, amount)
+end)
+
+RegisterNetEvent('consumables:server:setThirst', function (amount)
+    setThirst(source, amount)
+end)
+
+RegisterNetEvent('consumables:server:setHunger', function (amount)
+    setHunger(source, amount)
+end)
```

Comment:
The goal isn't to set a new value, it's to add to the current value

Reply:
Function should describe what it does, not what it gets. qbx_core gives the full value, which replaces the old thirst or water level.
That's what line
`local sustenance = (playerState.hunger or 0) + math.random(params.min, params.max)`
does. AddSomething is linked to the old value, SetSomething sets the new value with no relation to the old value.